src/opnsense/mvc/app/views/OPNsense/Firewall/alias_util.volt

```diff
@@ -123,6 +201,9 @@
                       <button class="btn btn-default pull-right" id="update_bogons"><i id="update_bogons_progress" class=""></i>
                         {{ lang._('Update bogons') }}
                       </button>
+                      <button class="btn btn-default pull-right" id="where_used">
+                          {{ lang._('Where used') }}
```

Comment:
We need to severely tweak this wording. Maybe "Find references" ?

Reply:
Enclosed a screenshot how Sophos does this:

![image](https://user-images.githubusercontent.com/20438689/49086513-fcf6af80-f254-11e8-8d92-4eaa53198368.png)

Reply:
"Where used" was my idea, but maybe an icon with a tooltip "Find references" would be better

Reply:
Same for Cisco:

![image](https://user-images.githubusercontent.com/20438689/49086619-45ae6880-f255-11e8-9a68-f9d61fcb94fb.png)

Reply:
@fichtner @mimugmail could we go for an hourglass + "Where used" then?

Reply:
@AdSchellevis I think you meant "magnifying glass" (vergrootglas), rather than "hourglass" (zandloper).

Reply:
oops, my mistake indeed

Reply:
OK, updated the PR.

Reply:
This is an opportunity to be better than Cisco? Why not use it ;)